### ISSUE TITLE
picture-elements, service-button not calling service correctly

### ISSUE THREAD

**Issue by ukmgranger** [home-assistant/home-assistant-polymer#2239]
**Home Assistant release with the issue:**
83.3

**Last working Home Assistant release (if known):**
N/A

**Browser and Operating System:**
all browsers

**Description of problem:**
I have added a service-button in lovelace, but when i press it in the front end, instead of toggling the single automation, it toggles all automations.

The toggling of the automation works in the dev-service page.
There is also a 'type: image' within the same picture element that toggles the automation on/off correctly.

code:

```
              - type: picture-elements
                image: /local/rooms/icons/tileBlank3.png
                elements:
                  - type: image
                    tap_action: toggle
                    entity: automation.wifi_away_mart
                    state_image:
                      off: /local/rooms/icons/tileBlank3.png
                      on: /local/rooms/icons/tileBlank3.png
                    style:
                      top: 50%
                      left: 50%
                      width: 100%
                      background: '#f2f4f4'
    
                  - type: image
                    title: test
                    tap_action: toggle
                    entity: automation.wifi_away_mart
                    state_image:
                      "on": /local/rooms/icons/shield-gray.png
                      "off": /local/rooms/icons/shield-blue.png
                    style:
                      top: 30%
                      left: 50%
                      width: 30%
                  - type: service-button
                    title: Guest Mode
                    service: automation.trigger
                    service_data:
                      entity_id: automation.wifi_away_disable
                    style:
                      top: 75%
                      left: 50%

```

**Javascript errors shown in the web inspector (if applicable):**
```

```

**Additional information:**

**Comment by iantrich** [comment]
Let me know if you're still having this issue on 0.84.1 when you upgrade

**Comment by ukmgranger** [comment]
> Let me know if you're still having this issue on 0.84.1 when you upgrade
Just updated to 0.84.1 and getting exactly the same issue.

The automation_toggle is still toggling all automations.  :(

My code:

```
                  - type: service-button
                    title: Guest Mode
                    service: automation.toggle
                    service_data:
                      entity_id: automation.wifi_away_mart
                    style:
                      top: 75%
                      left: 50%
```

**Comment by ukmgranger** [comment]
This is still not working in 0.84.2

**Comment by balloob** [comment]
Fixed in 0.84.3
